packages/nextjs-demo/src/app/agent/page.tsx

```diff
@@ -226,6 +258,13 @@ const ButtonComponent: React.FC<{ onClick: () => void; disabled: boolean; childr
   </button>
 );
 
+const LatencyComponent: React.FC<{ name: string; latency: number }> = ({ name, latency }) => (
```

Comment:
```suggestion
const Latency: React.FC<{ name: string; latency: number }> = ({ name, latency }) => (
```

Just like we wouldn't call it `namesArray` or `indexNumber`.

Reply:
I knew you were going to say this